### ISSUE TITLE
More info on the arguments please

### ISSUE THREAD

**Issue by Aneliram** [mpillar/java-emotion-recognizer#5]
I am trying to understand how the code works to recognize images. I have many questions but the main one at the moment is what are these arguments for:

arg1 = num_network_nodes (int)
arg2 = num_features (int)

What exactly do they do when it comes to kohonen or backprop?

Edit: I think i understood what they are for, the problem still remains though. Every time i run the train set i get completely different results. Same train set, same test set, same arguments -> COMPLETELY different results.
